_posts/2023-08-07-july-rubygems-updates.md

```diff
@@ -0,0 +1,87 @@
+---
+title: July 2023 RubyGems Updates
+layout: post
+author: Gift Egwuenu
+author_email: laurandidi21@gmail.com
+---
+
+Welcome to the RubyGems monthly update! As part of our efforts at Ruby Central, we publish a recap of the work that we’ve done the previous month. Read on to find out what updates were made to RubyGems and RubyGems.org in July.
+
+## RubyGems News
+
+This month in RubyGems, we released [3.4.16](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3416--2023-07-10), [3.4.17](https://github.com/rubygems/rubygems/blob/master/CHANGELOG.md#3417--2023-07-14) and Bundler [2.4.16](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2416-july-10-2023), [2.4.17](https://github.com/rubygems/rubygems/blob/master/bundler/CHANGELOG.md#2417-july-14-2023). 
+
+In RubyGems, we improved certain gem install invocations that had gotten slower since the dependencies API was phased out. In Bundler, the releases included several improvements such as fixes related to locked platforms, improvements to make sections related to git gems stable, and some other regression fixes.
+
+Some other improvements that landed into our repo this month but that are not included in the above releases are:
+
+- improved RubyGems `require` - [#6827](https://github.com/rubygems/rubygems/pull/6827).
+- improvements related to the Security Devices support feature - [#6774](https://github.com/rubygems/rubygems/pull/6774).
+- updated Magnus version in Rust extension gem template - [#6843](https://github.com/rubygems/rubygems/pull/6843).
+- removed side effects when unmarshaling old `Gem::Specification` - [#6825](https://github.com/rubygems/rubygems/pull/6825).
+- used `File::NULL `instead of hard coded null device names - [#6809](https://github.com/rubygems/rubygems/pull/6809).
+- added Automatiek improvements - [#6788](https://github.com/rubygems/rubygems/pull/6788).
+- fixed another incorrect removal of "Ruby" platform from lockfile when changing path sources - [#6784](https://github.com/rubygems/rubygems/pull/6784).
+- fixed git source lockfile instability - [#6786](https://github.com/rubygems/rubygems/pull/6786).
+- excluded Bundler from missing locked dependencies check - [#6792](https://github.com/rubygems/rubygems/pull/6792).
```

Comment:
Sorry for the back and forth @lauragift21. Actually last three entries were included in the above releases, so not necessary to list here.

And I actually meant to link to a couple of them in the second paragraph:

`In Bundler, the above releases include several improvements such as [fixes](https://github.com/rubygems/rubygems/pull/6784) related to locked platforms, [improvements](https://github.com/rubygems/rubygems/pull/6786) to make sections related to git gems stable, and some other regression fixes.`

Reply:
No problem at all, I'll update the post.